website/src/components/SequenceDetailsPage/SequencesDataTableTitle.astro

```diff
@@ -29,16 +29,33 @@ const { sequenceEntryHistory, accessionVersion, showFastaDownload } = Astro.prop
             )
         }
         {
-            showFastaDownload && (
-                <a
-                    class='sm:outlineButton inline-block'
-                    href={routes.sequencesFastaPage(accessionVersion) + '?download'}
-                >
-                    <span class='hidden sm:inline'>Download FASTA</span>
-                    <span class='sm:hidden inline text-xl'>
+            showDownload && (
+                <div class='sm:outlineButton inline-block dropdown dropdown-hover dropdown-end'>
+                    <span tabindex='0' class='hidden sm:inline cursor-pointer'>
+                        Download
```

Comment:
I think styling this as a button but it acting as a dropdown isn't great in terms of [affordances](https://en.wikipedia.org/wiki/Affordance). I (genuinely) find myself not expecting things to pop up. A few options:
- have two buttons, one for each action. I actually think this could be quite good as these are really common actions and we may not want to hide them behind a popover
- maybe use the same style as the button in the modal (which is a bit less buttony) - but I think this would be my lower preference

Reply:
Good feedback on this, thanks. I agree that the button looks like it shouldn't pop up. Maybe a little triangle could also make it more obvious. I don't quite understand the second option, can you clarify? For the 1st option (which I also considered) I didn't see a good way to do it on mobile, because on mobile the text turns into an icon, and then we'd need two icons? or we change the whole control on mobile entirely.

Maybe we can jump on a call to design something together before I fiddle with the code again.

Reply:
Next morning, I was quite fond of the idea with the little triangle, what do you think? 

<img width="158" alt="image" src="https://github.com/user-attachments/assets/a54f689f-f546-4c35-8860-20f09765813c">

Reply:
I've pushed the change for now! [b20b812](https://github.com/loculus-project/loculus/pull/3003/commits/b20b8123ab2e84c6e367d5f650b356342a6f1086)

Reply:
I agree - this seems nice!